src/components/Header/Header.component.jsx

```diff
@@ -5,14 +5,14 @@ import Search from '../Search';
 import ThemeToggle from '../ThemeToggle';
 import LoginAvatar from '../LoginAvatar';
 
-const Header = () => (
+const Header = (props) => (
```

Comment:
You could use destructuring here like:
```suggestion
const Header = ({ onSubmit ) => (
```

Then use it over the Search component